commit 54114994f4de7e8076fc250e44501e55e19b75b5
Author: Akinobu Mita <akinobu.mita@gmail.com>
Date:   Sun Jul 15 23:40:23 2007 -0700

    fault-injection: add min-order parameter to fail_page_alloc
    
    Limiting smaller allocation failures by fault injection helps to find real
    possible bugs.  Because higher order allocations are likely to fail and
    zero-order allocations are not likely to fail.
    
    This patch adds min-order parameter to fail_page_alloc.  It specifies the
    minimum page allocation order to be injected failures.
    
    Signed-off-by: Akinobu Mita <akinobu.mita@gmail.com>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

diff --git a/Documentation/fault-injection/fault-injection.txt b/Documentation/fault-injection/fault-injection.txt
index b7ca560b9340..36ece095ff17 100644
--- a/Documentation/fault-injection/fault-injection.txt
+++ b/Documentation/fault-injection/fault-injection.txt
@@ -103,6 +103,11 @@ configuration of fault-injection capabilities.
 	default is 'N', setting it to 'Y' will inject failures
 	only into non-sleep allocations (GFP_ATOMIC allocations).
 
+- /debug/fail_page_alloc/min-order:
+
+	specifies the minimum page allocation order to be injected
+	failures.
+
 o Boot option
 
 In order to inject faults while debugfs is not available (early boot time),